docs/core/project-sdk/overview.md

```diff
@@ -4,6 +4,7 @@ titleSuffix: ""
 description: Learn about the .NET project SDKs.
```

Comment:
We should add a line with link from the includes/excludes section of this file to the corresponding wpf/winforms sections.